packages/core/onClickOutside/index.ts

```diff
@@ -21,6 +22,11 @@ export interface OnClickOutsideOptions extends ConfigurableWindow {
    * @default false
    */
   detectIframe?: boolean
+  /**
+   * The name of the event to listen for.
+   * @default 'click'
+   */
+  eventName?: WindowEventName
```

Comment:
We could have an arrary for it, like
https://github.com/vueuse/vueuse/blob/5dad94ba2af7e16c685e27c4dc56335193f6a4fb/packages/core/useIdle/index.ts#L14-L19